### PR TITLE
added LU/de_genelogia_con_culaind.ttl

### PR DESCRIPTION
**Pull request by cyocum**
Added the file.

### CODE REVIEW

File: LU/de_genelogia_con_culaind.ttl [hunk 1]
```diff
@@ -0,0 +1,353 @@
...
+    a foaf:Person;
+    irishRel:genName "Íareóil Fátha";
+    irishRel:nomName "Íareól Fáth";
+    rel:childOf <#Erimoin>.
+
+<#Erimoin>
+    a foaf:Person;
+    irishRel:genName "Erimoin";
+    irishRel:nomName "Erimon";
+    rel:childOf <#MiledEspain>.
```
Comment (Eystein-Thanisch) [inline LU/de_genelogia_con_culaind.ttl]: owl:sameAs <http://example.com/LL/comuammand_na_n-genelach.ttl#Erímon>

File: LU/de_genelogia_con_culaind.ttl [hunk 2]
```diff
@@ -0,0 +1,353 @@
...
+    a foaf:Person;
+    irishRel:genName "Ethrióil";
+    irishRel:nomName "Ethriól";
+    rel:childOf <#ÍareóilFátha>.
+
+<#ÍareóilFátha>
+    a foaf:Person;
+    irishRel:genName "Íareóil Fátha";
+    irishRel:nomName "Íareól Fáth";
+    rel:childOf <#Erimoin>.
```
Comment (Eystein-Thanisch) [inline LU/de_genelogia_con_culaind.ttl]: owl:sameAs <http://example.com/LL/lagin.ttl#IriailFatha>

File: LU/de_genelogia_con_culaind.ttl [hunk 3]
```diff
@@ -0,0 +1,353 @@
...
+    irishRel:nomName "Tigerndmas";
+    rel:childOf <#Follaig>;
+    owl:sameAs <http://example.com/LL/dáil_araide.ttl#Thigernmais>.
+
+<#Follaig>
+    a foaf:Person;
+    irishRel:genName "Follaig";
+    irishRel:nomName "Follach";
+    rel:childOf <#Ethrióil>;
+    owl:sameAs <http://example.com/LL/dáil_araide.ttl#Fhallaig>.
```
Comment (Eystein-Thanisch) [inline LU/de_genelogia_con_culaind.ttl]: owl:sameAs http://example.com/LL/lagin.ttl#Fhallaig>

File: LU/de_genelogia_con_culaind.ttl [hunk 4]
```diff
@@ -0,0 +1,353 @@
...
+    irishRel:genName "Follaig";
+    irishRel:nomName "Follach";
+    rel:childOf <#Ethrióil>;
+    owl:sameAs <http://example.com/LL/dáil_araide.ttl#Fhallaig>.    
+
+<#Ethrióil>
+    a foaf:Person;
+    irishRel:genName "Ethrióil";
+    irishRel:nomName "Ethriól";
+    rel:childOf <#ÍareóilFátha>.
```
Comment (Eystein-Thanisch) [inline LU/de_genelogia_con_culaind.ttl]: owl:sameAs http://example.com/LL/lagin.ttl#Ethriuil

**Comment by cyocum** [comment]
Great thanks! I will make the changes later today.

**Comment by cyocum** [comment]
Ok, I have made the changes.  Let me know if you are happy and I will merge.

**Pull request merged**